apps/web/components/organisms/protectedComponent/protectedComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactNode } from 'react';
+import { useUser } from '@auth0/nextjs-auth0';
+import { Card, Heading, Link } from 'ui';
+
+type ProtectedComponentProps = {
+	info?: string;
+	children: ReactNode;
```

Comment:
Maybe adding an optional slot for a component that should be displayed in case user is not logged in?